website/docs/d/cen_bandwidth_limits.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: "alicloud"
+page_title: "Alicloud: alicloud_cen_bandwidth_limits"
+sidebar_current: "docs-alicloud-datasource-cen-bandwidth-limits"
+description: |-
+    Provides a list of CEN Inter Region Bandwidth Limits owned by an Alibaba Cloud account.
+---
+
+# alicloud\_cen\_bandwidth\_limits
+
+This data source provides CEN Inter Region Bandwidth Limits available to the user.
```

Comment:
What is `Inter Region`?

Reply:
It means cross-region.  It seems redundant here. I'll remove it .